spec/integration/shared_examples/rack_examples.rb

```diff
@@ -101,6 +101,18 @@ def wait_for_a_request_with_body(body)
       it "features userAgent" do
         wait_for_a_request_with_body(/"context":{.*"userAgent":"Bot".*}/)
       end
+
+      it "features referer" do
+        wait_for_a_request_with_body(/"context":{.*"referer":"bingo.com".*}/)
+      end
+
+      it "contains HTTP headers" do
```

Comment:
This is an example for an error in someone else's app. We simply ensure here that we are able to report headers if there were any. We already check our own content type when we send a request to our server. We have that test in airbrake-ruby.